lib/matplotlib/patches.py

```diff
@@ -1863,12 +1863,11 @@ def _pprint_styles(_styles):
     _table = [["Class", "Name", "Attrs"]]
 
     for name, cls in sorted(_styles.items()):
-        spec = inspect.getfullargspec(cls.__init__)
-        if spec.defaults:
-            argstr = ", ".join(map(
-                "{}={}".format, spec.args[-len(spec.defaults):], spec.defaults
-            ))
-        else:
+        sig = inspect.signature(cls.__init__)
+        argstr = ", ".join(
+            f"{p.name}={p.default}"
```

Comment:
Actually that's just `str(p)` (which will print out as `name=default` in presence of a default, isn't the stdlib wonderful? :p)

Reply:
"if something seems like it is too hard, it probably is"